packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md

```diff
@@ -0,0 +1,162 @@
+Use the Devices SDK to add custom push verification functionality to your Android app.
+
+See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/android/main/) for general information about mobile authentication at Okta.
+
+> **Note:** The following sample code assumes that suspend functions are called in a coroutine scope. See [Kotlin Coroutines](https://kotlinlang.org/docs/coroutines-overview.html) for more information.
+
+<!-- Data flow DIAGRAM -->
+
+### Install the dependency
+
+Add the Okta Authenticator SDK dependency to your `build.gradle` file:
+
+```kotlin
+implementation("com.okta.devices:devices-push:$okta.sdk.version")
+```
+
+where `$okta.sdk.version` is the latest release version. See [Release Status]() for the latest Okta Devices SDK version.
+
+### Initialize the client
+
+Create the SDK object to work with your Okta authenticator configuration. Use the `PushAuthenticatorBuilder` to create an authenticator with your application configuration:
+
+```kotlin
+val authenticator: PushAuthenticator = PushAuthenticatorBuilder.create(
+    ApplicationConfig(context, appName = "MyApp", appVersion = BuildConfig.VERSION_NAME)
+)
+```
```

Comment:
I've added the following in the README. 
```
val authenticator: PushAuthenticator = PushAuthenticatorBuilder.create(
    ApplicationConfig(context, appName = "MyApp", appVersion = BuildConfig.VERSION_NAME)
) {
    passphrase = "SecretPassphrase".toByteArray() // Secret must be stored securely 
}.getOrThrow()
```
If a passphrase isn't provided, then the Devices SDK data will not be encrypted. It is up to you to secure the passphrase.

If a passphrase isn't provided, then the Devices SDK data will not be encrypted. It is up to you to secure the passphrase.

Reply:
@FeiChen-okta Thanks! How do they secure the passphrase?

Reply:
Thank you, @FeiChen-okta ! Added.